core/templates/dev/head/css/oppia.css

```diff
@@ -4115,6 +4117,7 @@ md-card.preview-conversation-skin-supplemental-card {
   float: right;
   font-family: "Capriola", "Roboto", Arial, sans-serif;
   padding: 10px;
+  padding-bottom:35px;
```

Comment:
nit: add one space after colon.

It might be neater to combine this and the above line, viz.:

    padding: 10px 10px 35px 10px